store/models/signature.go

```diff
@@ -0,0 +1,86 @@
+package models
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+const (
+	// SignatureLength is the length of the signature in bytes: v = 1, r = 32, s
+	// = 32; v + r + s = 65
+	SignatureLength = 65
```

Comment:
This is not always guaranteed to be 65. V can be more than one byte. For instance, Geth private chains(often used for development) have a [chain ID of 1337](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids), making the V values 2709 or 2710. That's 2 bytes, and I don't think there's actually a limit. I think it'd be better to let Signature Length be.

Reply:
That's good to know, suggest we wait to encounter those before we fix it?

Reply:
Ok. I think we can wait, because I think that personal_sign doesn’t actually use a chain ID. Also, as you pointed out earlier ecrecover only allows v to be one byte.